### ISSUE TITLE
[v2] propel:migration:diff ignores "scale" in decimal column definition

### ISSUE THREAD

**Issue by bergcoder** [propelorm/PropelBundle#320]
After adding a decimal column (10,2) to the schema

`<column name="my_new_column" type="DECIMAL" size="10" scale="2" required="false" />`

the decimals are missing in the generated migration.

    ALTER TABLE `my_table`
        ADD
    (
        `my_new_column` DECIMAL
    );

**Comment by marcj** [comment]
Yeah, because they are the default value.

**Comment by marcj** [comment]
Obviously, the `scale` value is not always the default, although `10` for the size is correct.
